include/EVT/io/PWM.hpp

```diff
@@ -25,21 +25,21 @@ class PWM {
      *
      * @param[in] dutyCycle Duty cycle to set the pin to.
      */
-    virtual void setDutyCycle(float dutyCycle) = 0;
+    virtual void setDutyCycle(uint32_t dutyCycle) = 0;
 
     /**
      * Set the period for the PWM in seconds.
```

Comment:
Will probably need to change this comment to reflect the new units. True for other changed methods as well